tools/archive/templates/archive/issue.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Issue {{number}}</title>
+    <link href="https://webcompat.com/dist/webcompat.css" type="text/css" rel="stylesheet">
+    <style>table, td, th { border: 1px black solid; border-collapse: collapse;}</style>
+  </head>
+  <body id="body-webcompat">
```

Comment:
nit: I know the template will evolve, but here we're doing a dash and in the other classes we're using an underscore. Probably good to be consistent with however we currently do CSS for the rest of the site.

Reply:
Maybe I'm missing what you meant, but I used `body-webcompat` because

https://github.com/webcompat/webcompat.com/blob/a0becc68ccc5f30ca67383ec2adb712639d502d6/webcompat/templates/layout.html#L16

Reply:
Ah no understood.
For the other classes in this template!

Reply:
Yep, sorry for the confusion.